src/Components/TodoList.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Todo from "./Todo";
+
+// Todo List
+const TodoList = ({ todoList, remove, edit }) => {
+  const todoNode = todoList.map(todo => {
+    return <Todo todo={todo} key={todo.id} remove={remove} edit={edit} />;
```

Comment:
component name starting with small letters might get considered as default html tags instead of react component by JSX

keys are needed to uniquely identify the list items by react. If we don't pass them, it will create slow rendering issues for cases where elements are inserted deleted in list making the full tree re render